pkg/network/driver_interface.go

```diff
@@ -469,21 +520,28 @@ func (dh *DriverHandle) getStatsForHandle() (map[string]int64, error) {
 		}, nil
 	// A FlowHandle handle returns the flow stats specific to this handle
 	case FlowHandle:
+		if dh.lastNumFlowsMissed < uint64(stats.handle.flow_stats.num_flows_missed_max_exceeded) {
+			log.Warnf("Flows missed due to maximum flow limit. %v", stats.handle.flow_stats.num_flows_missed_max_exceeded)
+		}
+		dh.lastNumFlowsMissed = uint64(stats.handle.flow_stats.num_flows_missed_max_exceeded)
```

Comment:
Out of curiosity: on the linux side I believe we pretty much always try to report stats as a delta since the last time the stats were queried, but based on the logic here is looks like the stats gathered by the driver monotonically increase over the lifespan of each handle. Is there a specific reason for that difference?

Reply:
The driver just does a monotonic increase because it's fast and easy (and doesn't then require any locking).
We (bryce and I) talked about putting the calculation in the system probe (basically just subtracting the pvs. value from the current value), which we could quite easily do.